docs/user_guide/bank/getting_started_user_management.md

```diff
@@ -9,14 +9,14 @@ To manage the rights for your Essentials Bank's Users:
 
 Click on the My Organisation view, then  scroll down to the bottom. There is a User administration section there.
 
-You can also manage the Users in your partners, see [Administering Partner Users](pm_partner_user_admin.md)
+You can also manage the Users in your Partners, see [Administering Partner Users](pm_partner_user_admin.md).
 
 
 ![navigation for Users in organization](images/getting_started/user_admin/gs_user_admin_navigation.png)
 
 ## Inviting new Users
 Clicking "Invite User to this Organization" will open a popup where you will enter the name and email of the user.  Once you click "Invite User", they will receive an email with a link to follow to set up their password. 
-Please note that this link expires. If they don't click on the link in time, the workaround is for them to go to the login screen (http::/human_essentials.app) and click on "Reset Password". 
+Please note that this link expires. If they don't click on the link in time, the workaround is for them to go to the login screen and click on "Reset Password". 
```

Comment:
nit…seems callout-able? (and without the intervening blank line, it just sits in the same paragraph as the prev line of text)

```suggestion
> [!TIP]
> This link expires! If they don't click on the link in time, the workaround is for them to go to the login screen and click on "Reset Password" to generate a new email with link.
```